src/main/java/org/ibp/api/brapi/v1/germplasm/GermplasmResourceBrapi.java

```diff
@@ -125,6 +125,76 @@
 
 	}
 
+	@ApiOperation(value = "Search germplasms", notes = "Replacement for germplasm-search in v1.3")
```

Comment:
Let's extract changes to this new endpoint to another ticket/PR:
https://ibplatform.atlassian.net/browse/IBP-3624